src/theme/icon.rs

```diff
@@ -405,7 +502,8 @@ impl IconTheme {
             ("rtf", "\u{f15c}"),             // ""
             ("ru", "\u{e21e}"),              // ""
             ("rubydoc", "\u{e73b}"),         // ""
-            ("s", "\u{e614}"),               // ""
+            ("s", "\u{f471}"),               // ""
+            ("snippets", "\u{e62b}"),        // ""
```

Comment:
Maybe we can give a more generic icon than vim icon for snippets.

Reply:
I am not sure what would best suite snippets. Just from a quick search I could not find anything that apt(other than maybe some generic file icons). That said, I don't think we should give it vim icon as the same name might be used by multiple editors.

Reply:
> I am not sure what would best suite snippets. Just from a quick search I could not find anything that apt(other than maybe some generic file icons). That said, I don't think we should give it vim icon as the same name might be used by multiple editors.

Ah, well, I'll remove it then